### ISSUE TITLE
A little help / documentation perhaps..

### ISSUE THREAD

**Issue by abclution** [FacetWP/github-updater-lite#13]
I have a small repo on github with some customization for my wordpress "functions.php" written as a plugin. I believe I have followed the instructions but if I make any commits and changes I don't get any autoupdates or prompts to do so. 

I have put the 
include( dirname( __FILE__ ) . '/github-updater.php' );
right near the top of
TKS-CustomFunctions.php

Have I integrated it wrongly?

Here is the repo
https://github.com/abclution/TKS-CustomFunctions

**Comment by mgibbs189** [comment]
This line:

```
GitHub Plugin URI: https://github.com/abclution/TKS-CustomFunctions.git
```

should look like this:

```
GitHub URI: abclution/TKS-CustomFunctions
```

Also remember that updates are only checked every 12 hours. To force an update, you'll need to remove options starting with `ghu_` from the wp_options table.

**Comment by abclution** [comment]
Hi, 
Got it working thanks.
Pretty cool.

Some things to add to documentation, correct me if I am wrong.
I was expecting it to update based on commit numbers.
Then I looked at the code and saw it checks for "releases"
Created a higher version release and next check interval it showed up. :)

I also adjusted the release interval check instead of going through the trouble of dropping the table options. :)

Thank you!
